components/Common/Pagination/__tests__/index.test.mjs

```diff
@@ -0,0 +1,175 @@
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+
+import Pagination from '@/components/Common/Pagination';
+
+jest.mock('next/router', () => ({
```

Comment:
Can we add this as a sort of global mock? I bet somewhere this is already being mocked, if not, probably suits making a global mock 👀

Reply:
That's a great idea. I have created a `nextJsRouter.mjs` mock within a `__mocks__` directory, and imported it within `jest.setup.mjs`. I have also removed the existing `next/router` mocks in test files, and made sure the tests are still successful.